### ISSUE TITLE
JavaScript Course: Update codeblock delimiters per layout style guide (computer_science && testing_javascript)

### ISSUE THREAD

**Issue by aconstas** [TheOdinProject/curriculum#26962]
### Checks

- [X] This is not a duplicate of an existing issue (please have a look through our [open issues list](https://github.com/TheOdinProject/curriculum/issues) to make sure)
- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [X] Would you like to work on this issue?

### Describe your suggestion

Codeblocks should be updated to align with new changes in our [layout style guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

In the lessons named in this issue, codeblocks will be opened by a set of 3 tildes (the ~ character) and then closed by another set of 3 tildes. To align with the style guide, these tilde characters need to change to backticks (the ` character). Be sure to refer to [the codeblocks section](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md#codeblocks) of the style guide for an example of what this should look like.
## Guidance:
- Comment on this issue to get a lesson assigned to you. You can also ask for a specific one
- Please wait until you are assigned by a maintainer before starting to work on this issue to avoid multiple people working on the same lessons
- Make sure to read the [contributing guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md) before starting your first contribution
- Please add the following to your PR in the Issues section: Related to #xxxxx, since this issue is divided into sections, individual PRs should NOT INCLUDE the closes keyword.

## Lessons to update
**Computer Science**
- [x] @Ahm3rIjaz [Project: Binary Search Trees](https://www.theodinproject.com/lessons/javascript-binary-search-trees)
- [x] @riarsdev [Space Complexity](https://www.theodinproject.com/lessons/javascript-space-complexity)

**Testing JavaScript**
- [x] @VMadhuranga [More Testing](https://www.theodinproject.com/lessons/node-path-javascript-more-testing)
- [ ] @janaiscoding [Project: Testing Practice](https://www.theodinproject.com/lessons/node-path-javascript-testing-practice)

@JoshDevHub

### Path

Node / JS

### Lesson Url

javascript/computer_science && javascript/testing_javascript

**Comment by Ahm3rIjaz** [comment]
Hi, I want to work on this issue.

**Comment by riarsdev** [comment]
I'd be happy to contribute. Count me in!

**Comment by VMadhuranga** [comment]
I would also like to help

**Comment by JoshDevHub** [comment]
Thank you all for volunteering!

@Ahm3rIjaz I've assigned you the Binary Trees project

@riarsdev I've assigned you the Space Complexity lesson

@VMadhuranga I've assigned you the More Testing lesson

**Comment by janaiscoding** [comment]
Hello, I would like to help with this issue!

**Comment by JoshDevHub** [comment]
@janaiscoding Thank you for volunteering

I've assigned you the Testing Practice project

**Comment by JoshDevHub** [comment]
Thanks for the contributions everyone.
